command/plugins_remove.go

```diff
@@ -34,6 +38,13 @@ Usage: packer plugins remove <plugin> [<version constraint>]
   When the version is omitted all installed versions will be removed.
 
   Ex: packer plugins remove github.com/hashicorp/happycloud v1.2.3
+
+Alternatively, the plugin source can be a path to an installed plugin.
+This syntax does not however allow for version specifiers.
+
+  Ex: packer plugins remove ~/.config/plugins/github.com/hashicorp/happycloud/packer-plugin-happycloud_v1.0.0_x5.0_linux_amd64
+
+Note: in order for this plugin removal to work, the plugin's path should be absolute. Plugin binaries outside of the plugin directory will not be removed.
```

Comment:
The alignment is slightly off. 
```
Usage: packer plugins remove <plugin> [<version constraint>]

  This command will remove all Packer plugins matching the version constraint
  for the current OS and architecture.
  When the version is omitted all installed versions will be removed.

  Ex: packer plugins remove github.com/hashicorp/happycloud v1.2.3

Alternatively, the plugin source can be a path to an installed plugin.
This syntax does not however allow for version specifiers.

  Ex: packer plugins remove ~/.config/plugins/github.com/hashicorp/happycloud/packer-plugin-happycloud_v1.0.0_x5.0_linux_amd64

Note: in order for this plugin removal to work, the plugin's path should be absolute. Plugin binaries outside of the plugin directory will not be removed.
```

How do you feel about changing the text up a little bit. 
```
Usage: packer plugins remove <plugin> [<version constraint>]

  This command will remove one or more installed Packer plugins. 
  
  To remove a plugin matching a version contraint for the current OS and architecture.
  
      packer plugins remove github.com/hashicorp/happycloud v1.2.3

  To remove all versions of a plugin for the current OS and architecture omit the version constraint.  
      packer plugins remove github.com/hashicorp/happycloud 

  To remove a single plugin binary from the Packer plugin directory specify the absolute path to
  an installed binary. This syntax does not allow for version matching. 

      packer plugins remove ~/.config/plugins/github.com/hashicorp/happycloud/packer-plugin-happycloud_v1.0.0_x5.0_linux_amd64

```